extensions/observability-devservices/deployment/src/main/java/io/quarkus/observability/deployment/devui/ObservabilityDevUIProcessor.java

```diff
@@ -26,45 +26,46 @@ public class ObservabilityDevUIProcessor {
     @BuildStep(onlyIf = IsDevelopment.class)
     void createVersion(BuildProducer<CardPageBuildItem> cardPageBuildItemBuildProducer,
             BuildProducer<FooterPageBuildItem> footerProducer,
-            Optional<ObservabilityDevServicesConfigBuildItem> configProps) {
+            List<ObservabilityDevServicesConfigBuildItem> configProps) {
 
-        // LGTM
-        if (configProps.isPresent()) {
-            Map<String, String> runtimeConfig = configProps.get().getConfig();
-            String grafanaUrl = runtimeConfig.getOrDefault("grafana.endpoint", "");
+        if (configProps != null) {
+            configProps.forEach(cp -> {
```

Comment:
No need to change it here but we should really avoid using lambdas when they have no value. In this case a for loop would have been better.

Also `configProps` should never be null.

Reply:
Fixed